src/StoreKeeper/WooCommerce/B2C/PaymentGateway/PaymentGateway.php

```diff
@@ -370,11 +362,17 @@ public static function refundExists($orderId, $refundId): ?bool
 
         // Getting the results and getting the first one.
         $results = $wpdb->get_results($sql, ARRAY_A);
-        if (!empty($results)) {
-            $payment_id = array_shift($results)['sk_refund_id'];
+
+        if (empty($results)) {
+            return false;
         }
 
-        return $payment_id;
+        $refund = array_shift($results);
```

Comment:
why only first?

Reply:
Changed this to check all refund records that matched order id and refund id.